File: cpp/non-graphical/random_integer_generator.cpp
#include <random>
#include <fstream>
#include "random_integer_generator.h"
using namespace std;

random_integer_generator::random_integer_generator(long n){
  num_to_generate = n;
  generate();
}

void random_integer_generator::generate(){
  ofstream out_file;
  out_file.open("rand_ints.txt");

  random_device rd;
  mt19937 gen(rd());
  uniform_int_distribution<> dis(1, 1000000000);

  for (int n=0; n<num_to_generate; ++n){
      out_file << dis(gen) << "\n";
  }
  out_file.close();
}

File: python/non-graphical/random_integer_generator.py
#!/usr/bin/env python3
import random

class generator(object):
	def __init__(self, number):
		self.num_numbers = number
		self.generate()

	def generate(self):
		with open("rand_ints.txt", mode='w') as WRITE_FILE:
		    counter = 0
		    while(counter < self.num_numbers):
		        WRITE_FILE.write(str(random.randint(2,1000000000)) + '\n')
		        counter+=1
		WRITE_FILE.close()
